lib/winston-modern-syslog.js

```diff
@@ -20,13 +21,27 @@ var ModernSyslog = module.exports = winston.transports.ModernSyslog = function (
     }
 
     // open the stream
-    syslog.open(this.label, syslogOptions);
+    syslog.open(this.label, syslogOptions, this.facility);
 };
 
 util.inherits(ModernSyslog, winston.Transport);
 
 ModernSyslog.prototype.name = 'ModernSyslog';
 
+ModernSyslog.prototype.levels = {
```

Comment:
I know these integers are syslog standard, but I'd prefer to reference the `modern-syslog` level constants here, e.g. `'emerg': syslog.level.LOG_EMERG` etc.